docs/kubebuilder_v0_v1_difference.md

```diff
@@ -13,7 +13,7 @@ Kubebuilder 1.0 adds a new flag `--project-version`, it accepts two different va
   bin/controller-manager --kubeconfig ~/.kube/config
 
   kubectl apply -f hack/sample/<resource>.yaml
-  docker build -f Dockerfile.controller . -t <image:tag>
+  docker build -f Dockerfile.controller -t <image:tag> .
```

Comment:
Nit: revert this, since this is an old doc

Reply:
Done with 7769cd92 -> 3b13378b